code/game/objects/items/robot/cyborg_gripper.dm

```diff
@@ -1,10 +1,12 @@
 /*   CONTENTS:
-0. generic define gripper
-1. UNIVERSAL GRIPPER
-2. MEDICAL GRIPPER
-3. SERVICE GRIPPER
-4. MINING GRIPPER
-5. ENGINEERING GRIPPER
+0. Generic Gripper
+1. Hug Module
```

Comment:
```suggestion
```
this pr removes the hug module, no?

Reply:
Yes. I was just attempting to mark where in the code all those interactions start. Probably not actually necessary so I'll remove.